cmd/coco/reconcile/reconcile.go

```diff
@@ -0,0 +1,294 @@
+package reconcile
+
+import (
+	"context"
+	"fmt"
+	"os"
+	"strings"
+
+	"github.com/google/go-github/v51/github"
+	"golang.org/x/oauth2"
+)
+
+func StartReconcilition(sourceBranch string, targetBranch string, ownerName string, repoName string, dryRun bool) error {
```

Comment:
How about this name?
```suggestion
func Reconcile(sourceBranch string, targetBranch string, ownerName string, repoName string, dryRun bool) error {
```